src/main/scala/com/datastax/cdm/job/BasePartitionJob.scala

```diff
@@ -17,10 +17,22 @@ package com.datastax.cdm.job
 
 import java.util
 import scala.reflect.io.File
+import com.datastax.cdm.feature.TrackRun
+import com.datastax.cdm.properties.KnownProperties
 
 abstract class BasePartitionJob extends BaseJob[SplitPartitions.Partition] {
+  var trackRunFeature: TrackRun = _
+
   override def getParts(pieces: Int): util.Collection[SplitPartitions.Partition] = {
-    if (!File(this.partitionFileNameInput).exists) {
+    var keyspaceTable = propertyHelper.getString(KnownProperties.TARGET_KEYSPACE_TABLE)
```

Comment:
@pravinbhat since target keyspace.table is optional and will default to origin keyspace.table, this will break the job when the properties file doesn't specify it explicitly.